test/integration/setup.js

```diff
@@ -250,14 +282,16 @@ async function setup() {
   await ramm.changeDependentContractAddress();
   await ramm.connect(emergencyAdmin).setEmergencySwapPause(false);
 
-  await legacyPool.updateAddressParameters(toBytes8('SWP_OP'), swapOperator.address);
+  // Manually add pool assets that are not automatically added via LegacyPool constructor
   await legacyPool.addAsset(
     usdc.address,
     true,
     parseUnits('1000000', usdcDecimals),
     parseUnits('2000000', usdcDecimals),
     250,
   );
```

Comment:
i checked the [Pool.getAssets](https://etherscan.io/readContract?m=light&a=0xcafeaBED7e0653aFe9674A3ad862b78DB3F36e60&n=mainnet&v=0xcafeaBED7e0653aFe9674A3ad862b78DB3F36e60#readCollapse7) and it does not contain USDC as an asset.

I tried removing this line but it breaks `cover.setProducts`.

Should we bother aligning this with the deployed contract or is there no need?